catalogs/qhimm.xml

```diff
@@ -1341,17 +1341,17 @@ E.g., Bolt2 = Thundara</Description>
       <Author>Mewsin</Author>
       <Link>https://forums.qhimm.com/index.php?topic=13383.0</Link>
       <LatestVersion>
-		<Link>iros://Url/http$qhimm.7thheaven.rocks/Catalog 4.0/SpellTextures/Coin_Skill.7z</Link>
```

Comment:
I'd suggest not changing this URL

Reply:
I'm not gonna

Reply:
since these have been hosted on my behalf and are not actually an iro format yet, and i am new to using github, how do i then do this, the real change is to the xml file only

Reply:
i took your updated iros and uploaded them to the qhimm server. since these aren't mods that are going to be updated regularly, it's better to have them in a more reliable place.